portal/src/main/webapp/js/src/civic/civicservice.js

```diff
@@ -0,0 +1,166 @@
+
+function CivicService() {
+    
+    // All results are stored here
+    var civicGenes = {};
+    
+    function getCivicGenesBatch(ids) {
+        return $.ajax({
+            type: 'GET',
+            url: 'api-legacy/proxy/civic/genes/' + ids,
```

Comment:
Hi!

Would you mind providing an example request that's failing cross-origin? As far as I know CORS is already enabled on CIViC for all API requests (https://github.com/genome/civic-server/blob/master/app/controllers/concerns/api_cors.rb) and I'm seeing the expected headers in the API responses:

```
~/git/civic-server as acoffman at acoffman-macbook-pro
λ curl -v https://civic.genome.wustl.edu/api/genes/1
*   Trying 10.0.97.29...
* TCP_NODELAY set
* Connected to civic.genome.wustl.edu (10.0.97.29) port 443 (#0)
* TLS 1.2 connection using TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
* Server certificate: *.genome.wustl.edu
* Server certificate: thawte SSL CA - G2
* Server certificate: thawte Primary Root CA
> GET /api/genes/1 HTTP/1.1
> Host: civic.genome.wustl.edu
> User-Agent: curl/7.51.0
> Accept: */*
>
< HTTP/1.1 200 OK
< Date: Thu, 26 Jan 2017 15:50:20 GMT
< Server: Apache/2.2.22 (Ubuntu)
< X-Frame-Options: SAMEORIGIN
< X-XSS-Protection: 1; mode=block
< X-Content-Type-Options: nosniff
< Access-Control-Allow-Origin: *
< Access-Control-Allow-Methods: POST, GET, PUT, DELETE, OPTIONS
< Access-Control-Allow-Headers: Origin, Content-Type, Accept, Authorization, Token
< Access-Control-Max-Age: 1728000
```